services/movie_session.py

```diff
@@ -3,6 +3,14 @@
 from db.models import MovieSession
 
 
+def get_taken_seats(movie_session_id: int) -> list:
+    return list(
+        MovieSession.objects.get(id=movie_session_id).tickets.values(
```

Comment:
The `get` method does not handle the case where a `MovieSession` with the given ID does not exist. Consider using `get_object_or_404` or handling the `DoesNotExist` exception to provide a more user-friendly error message.